content/hardware/05.pro-solutions/solutions-and-kits/stella/tutorials/01.user-manual/content.md

```diff
@@ -282,27 +361,4146 @@ The `Nearby World` example demonstrates the core functionality of UWB technology
+4. Observe the LED feedback on both devices
+
+***If you don't see any measurements, verify that both devices are powered on and running their respective sketches. The system requires both the Controller and Controlee to be active for ranging to occur.***
+
+**Visual Feedback During Operation:**
+
+When the system is working correctly, both devices provide LED feedback:
+
+- **Arduino Stella LED:** Blinks faster as devices get closer (fast at 50 cm, slow at 300 cm)
+- **Portenta C33 Red LED:** Blinks continuously to show system is active
+- **Portenta C33 Blue LED:** OFF when connected, ON when connection is lost
+- **Portenta C33 Green LED:** ON when Stella is within 300 cm, OFF when further away
+
+**Serial Output Visualization:**
+
+The Serial Plotter provides real-time visualization of the distance measurements:
+
+![Serial Plotter showing real-time distance measurements and moving average](assets/twr-serial-plotter.gif)
+
+The graph shows two lines:
+
+- **Blue line:** Raw distance measurements in centimeters
+- **Red line:** Smoothed moving average for trend analysis
+
+Example readings: `Distance(cm):125,Average(cm):123` indicates the devices are approximately 1.25 meters apart.
+
+### Extending the Two-Way Ranging Example
+
+This enhanced example provides a foundation for more advanced applications:
+
+- **Access control systems**: Use the distance threshold to trigger door locks or security systems
+- **Safety zones**: Create multiple distance thresholds for industrial safety applications
+- **Data logging**: Add SD card storage to record distance measurements over time
+- **Multi-node networks**: Extend to support multiple tags communicating with one base station
+- **Position calculation**: Combine multiple base stations for 2D/3D positioning through triangulation
+- **Alert systems**: Add buzzer or additional LEDs for specific distance-based alerts
+
+The visual feedback and data processing capabilities demonstrated in this example can be adapted to various real-world applications requiring precise distance measurement and proximity detection.
+
+## Support
+
+If you encounter any issues or have questions while working with your Arduino Stella, we provide various support resources to help you find answers and solutions.
+
+### Help Center
+
+Explore our Help Center, which offers a comprehensive collection of articles and guides for Arduino boards and shields. The Help Center is designed to provide in-depth technical assistance and help you make the most of your device.
+
+- [Arduino Help Center](https://support.arduino.cc/hc/en-us)
+
+### Forum
+
+Join our community forum to connect with other Arduino Stella users, share your experiences, and ask questions. The Forum is an excellent place to learn from others, discuss issues, and discover new ideas and projects related to the Arduino Stella.
+
+- [Arduino Stella in the Arduino Forum](https://forum.arduino.cc/c/hardware/stella/91)
+
+### Contact Us
+

```

Comment:
I think there is a weird duplicate in the content, as I see it's repeating here again. Please check it out :)